manifests/master/pdf_stack.pp

```diff
@@ -1,7 +1,7 @@
 # Dependencies for PDF rendering
 class classroom::master::pdf_stack {
 
-  $enabled = $classroom::offline ? {
+  $enabled = defined('$classroom::offline') and $classroom::offline ? {
```

Comment:
Thanks @binford2k, I'll try that. Previously, I had just `include classroom` and that was causing duplication declaration errors in some of the tests. Need to remember that I can rewrite classes on the fly like your example.